cpp/CMakeLists.txt

```diff
@@ -53,3 +68,4 @@ if(CMAKE_EXPORT_COMPILE_COMMANDS)
     COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_BINARY_DIR}/compile_commands.json
             ${PROJECT_SOURCE_DIR})
 endif()
+]]
```

Comment:
should this be commented?

Reply:
I left some things in, but commented out for the purpose of not forgetting to include them in later PRs, but I could just go to the main branch to see what was there before. I'll just get rid of them.